.github/workflows/build_and_test.yml

```
@@ -5,26 +5,32 @@ on:
       matrix:
-        php-version: [ '7.4', '8.0', '8.1', '8.2', '8.3' ]
+        php-version: [ '7.4', '8.0', '8.1', '8.2', '8.3', '8.4' ]
 
     steps:
     - uses: actions/checkout@v3
 
+    - if: ${{ env.act }}
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: "${{ matrix.php-version }}"
+        tools: phpunit:${{ matrix.phpunit-versions }}
```

actionlint (1.7.4)

property "phpunit-versions" is not defined in object type {php-version: number}

(expression)